### ISSUE TITLE
Searching for activities of part-up get in loop when there are zero activities and user is not logged in

### ISSUE THREAD

**Issue by AnnotatedJS** [part-up/part-up#1304]
Searching for activities of part-up get in loop when there are zero activities and user is not logged in.
And the whole site does not function as intended.

Noticed this error while exploring part-up activities in the context of issue #1169

**Comment by noelheesen** [comment]
@ralphboeije I believe you are not allowed to see activities when you are not a partner? It then tries to get all the activities with an undefined userId.

**Comment by ralphboeije** [comment]
@noelheesen Yes, you are allowed to see activities, just not to add/contribute on.

**Comment by AnnotatedJS** [comment]
With searching for activities of a part-up I meant, clicking on the tab Activities of a part-up page.
